Gemfile

```diff
@@ -9,84 +9,77 @@ end
 
 gem 'rails', '~> 5.2.4'
 
-gem 'puma', '~> 4.3'
-gem 'pg', '~> 1.1'
+gem 'activerecord-session_store'
```

Comment:
We could just enable this in rubocop.yml, at the moment is disabled, and check it there are warnings.

```
 Bundler/OrderedGems:
    Enabled: true
```

Ideally put this piece of work into another PR

Reply:
I'll add it in, rebase and push.  Good suggestion.  I didn't realise rubocop offered this cop.